pkgs/applications/virtualization/libgovirt/default.nix

```diff
@@ -11,13 +11,13 @@
 
 stdenv.mkDerivation rec {
   pname = "libgovirt";
-  version = "0.3.8";
+  version = "0.3.9";
```

Comment:
https://gitlab.gnome.org/GNOME/libgovirt/-/compare/v0.3.8...v0.3.9

This update sounds not that trivial to me:

- 2decf236: "Remove autotools files"
- 67f5d178: "Port to rest-1.0 & soup-3.0"

Reply:
Changed to meson and built

```
nix-build -A libgovirt                                           Sat 05 Nov 2022 11:54:22 AM CST
/nix/store/i21jg13j71z5a9dy97gxwvhkac6i9lh9-libgovirt-0.3.9
```

Reply:
Since this is ported to libsoup-3, I think virt-viewer will be affected: https://gitlab.com/virt-viewer/virt-viewer/-/issues/88

We have three options: (I didn't really test):

- A. Drop the libgovirt bump
- B. Patch virt-viewer to use rest-1.0 (the [Arch](https://github.com/archlinux/svntogit-community/blob/packages/virt-viewer/trunk/rest.diff) way)
- C. Disable govirt for virt-viewer (the [Fedora](https://src.fedoraproject.org/rpms/virt-viewer/c/1c755670cc41e20ae28ff50f51d0e7162ffddf07?branch=rawhide) way)

Reply:
I will go with A), thanks a lot for looking into this.